_redirects

```diff
@@ -1,8 +1,5 @@
 # Feed
-/atom.xml /blog/feed.xml
```

Comment:
None of the `/blog/` bits in this file need to be changed as they are for legacy links from other websites to be redirected to the new uri syntax.

All that needs doing to the `_redirects` file is adding a redirect from the old feed uri to the new feed uri:

```
/blog/feed.xml /writing/feed.xml
```

Reply:
Awesome will do thanks for the input will push changes asap :)

Reply:
So I've reverted changes and squashed into one commit to be simpler :) @carbontwelve  pinging you for another quick review